tests/integration/fixtures/mini_sentry.py

```diff
@@ -316,9 +318,7 @@ def get_project_config():
         version = flask_request.args.get("version")
 
         if version == "4" and flask_request.json.get("global"):
-            # There are no fields in global config for now
-            global_ = {}
-
+            global_ = GLOBAL_CONFIG
```

Comment:
Why did we decide to make the global config a constant there, over e.g. returning it from a function?

Reply:
is it preferred to use a function? If it's totally static wouldn't it be good to just have it as a variable?

Reply:
Sounds good to me, no strong opinions on that. I think we should move it to `mini_sentry` though.